stable/concourse/values.yaml

```diff
@@ -665,6 +665,13 @@ web:
   #   - name: CONCOURSE_TSA_LOG_LEVEL
   #     value: "debug"
 
+  ## For managing where secrets should be mounted for the web agents
```

Comment:
why are you commenting these out and then using defaults in the templates ?

Reply:
Would a better approach be to remove the call to default in the templates and just set the default values here?

Reply:
sure sounds better